images/router/haproxy-base/Dockerfile

```diff
@@ -6,18 +6,11 @@
 #
 FROM openshift/origin-base
 
-RUN yum install -y gcc make openssl-devel pcre-devel tar wget socat && \
-    wget http://www.haproxy.org/download/1.5/src/haproxy-1.5.10.tar.gz  && \
-    tar xvzf haproxy-1.5.10.tar.gz && \
-    groupadd haproxy && \
-    useradd -g haproxy haproxy && \
-    cd haproxy-1.5.10 && make TARGET=linux2628 CPU=generic USE_PCRE=1 USE_OPENSSL=1 USE_ZLIB=1 && make install && \
-    cd .. && rm -rf haproxy-1.5.10 && \
+RUN yum -y install https://tdawson.fedorapeople.org/openshift/haproxy-1.5.4-2.el7.x86_64.rpm && \
     mkdir -p /usr/bin && \
     mkdir -p /var/lib/containers/router/{certs,cacerts} && \
```

Comment:
@tdawson This and line 19 look suspicious...  I wouldn't expect the default haproxy RPMs to have those directories.